### PR TITLE
[Beautification] Re-themed ENTIRE Project

### PR DESCRIPTION
**Pull request by MuhsinunC**
Title says it all. I've been using this for a while, and so now I decided to make it fit in with the rest of my website. Using these files, the project looks like it's part of my website software, NamelessMC. Don't worry, this doesn't require NamelessMC in order to run, it's just made so that it looks better conformed to run with NamelessMC. <3 :O :D

I'm planning on adding auto theme changing based on the NamelessMC theme thru a MySQL database :O :D

[Spigot MC Resource Posting (With NamelessMC addon capabilities)](https://www.spigotmc.org/resources/core-protect-2-web-interface-nameless-mc-addon.34171/)

Screenshots:
image
image
image

### CODE REVIEW

File: web/res/php/webtemplate.php [hunk 1]
```diff
@@ -27,28 +27,229 @@ public function head() {?>
   <title><?php echo $this->t;?></title>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css">
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.3/css/bootstrap.min.css" integrity="sha384-MIwDKRSSImVFAZCVLtU0LMDdON6KVCrZHyVQQj6e8wIEJkW4tvwqXrbMIya1vriY" crossorigin="anonymous">
+  <!-- CUSTOM THEMES HERE!!! -->
```
Comment (chuushi) [inline web/res/php/webtemplate.php:30]: Note that this project uses Bootstrap 4.0.0 elements. That's why the buttons look broken with these themes.

Comment (chuushi) [reply web/res/php/webtemplate.php:30]: Can we switch these files over with a CDN instead? Your changes bloats the project so much. ._. https://www.bootstrapcdn.com/bootswatch/

Comment (MuhsinunC) [reply web/res/php/webtemplate.php:30]: Yea sure! Feel free to change it to your liking! :D

Comment (MuhsinunC) [reply web/res/php/webtemplate.php:30]: Sorry, didn't know u were using Bootstrap 4.0.0 or that I was using an old version of the themes. My bad!

**Comment by chuushi** [comment]
I want to use different module implementation for NamelessMC.  The PR made some minor fundamental changes and added a lot of bloaty files in an inefficient way as your branch stands now.

I've made a new branch named 1.10.0-SNAPSHOT, so if there's anything you think you can add, you can make a PR towards there after I have it go one or two commits ahead of master branch. :)

I am going to try to turn many parts of the project into classes and object-orientate it, as it is currently written in and left in the same way I wrote this back in 2015, which is pretty hard to maintain.

**Comment by MuhsinunC** [comment]
Alr! Just hmu wen that other branch's ready for dev! I'll be here learning lmao

**Comment by MuhsinunC** [comment]
From now on, I'm making all of my stuff Material Design Lite themed, as that is much, much more professional imo & much prettier imo :D glhf!

**Pull request closed without merging**